CONTRIBUTING.md

```diff
@@ -132,7 +132,8 @@ For model named `<main_model>` the following files may be created:
 * `templates/<main_model>.xml`
 * `views/<main_model>.xml`
 
-For `controller`, the only file should be named `main.py`.
+For `controller`, if there is only file should be named `main.py`. If there
+are several controller methods you can split them in several files.
```

Comment:
"If there are several controller methods you can split them in several files." should be "If there are several controller classes or functions you can split them into several files."